set.mm

```diff
@@ -39903,13 +39903,13 @@ will do (e.g., ` O = (/) ` and ` T = { (/) } ` , see ~ 0nep0 ).
     raltpd.c $e |- ( ph -> C e. X ) $.
     $( Convert a quantification over a triple to a conjunction.  (Contributed
        by Thierry Arnoux, 8-Apr-2019.) $)
-    raltpd $p |- ( ph -> ( A. x e. { A , B , C } ps <-> ( ch /\ th /\ ta ) ) )
-      $=
-      ( wa wb ctp wral w3a an3andi a1i wcel cv expcom pm5.32d raltpg syl3anc c0
-      wceq wne tpnzd r19.28zv syl 3bitr2d bianabs bicomd ) ABFGHIUAZUBZCDEUCZAA
-      VCSZVBAVDVBAVDACSZADSZAESZUCZABSZFVAUBZAVBSZVDVHTAACDEUDUEAGJUFHKUFILUFVJ
-      VHTPQRVIVEVFVGFGHIJKLFUGZGUMZABCAVMBCTMUHUIVLHUMZABDAVNBDTNUHUIVLIUMZABEA
-      VOBETOUHUIUJUKAVAULUNVJVKTAGHIJPUOABFVAUPUQURUSUTUS $.
+    raltpd $p
+            |- ( ph -> ( A. x e. { A , B , C } ps <-> ( ch /\ th /\ ta ) ) ) $=
+  ( wa wb ctp wral w3a an3andi a1i wcel cv expcom pm5.32d raltpg syl3anc c0 wne
```

Comment:
Why has the indentation changed here?  This seems to be triggered by $= in the last position of a line.

Reply:
Like for the following cases, this is indeed a bug of the rewrapping caused by `$p` ending a line, see https://github.com/metamath/metamath-exe/issues/93
There is a suggestion to fix it by @savask, maybe you can check if it is correct. When this is fixed that the new version of metamath is used by the CI, then proofs should be rewrapped as usual. I think this is not a problem, and when it is possible to have the whole statement `|- ... $=` on a single line, it is more readable to do so.

Reply:
> Like for the following cases, this is indeed a bug of the rewrapping caused by `$p` ending a line, see [metamath/metamath-exe#93](https://github.com/metamath/metamath-exe/issues/93) There is a suggestion to fix it by @savask, maybe you can check if it is correct. When this is fixed that the new version of metamath is used by the CI, then proofs should be rewrapped as usual. I think this is not a problem, and when it is possible to have the whole statement `|- ... $=` on a single line, it is more readable to do so.

I do not agree that a whole (long) statement on a signle line is generally more readable. In the current example,

```
raltpd $p |- ( ph -> ( A. x e. { A , B , C } ps 
                       <-> ( ch /\ th /\ ta ) ) ) $=
```

would also be possible. At least, we should not force to squeeze whole statements into a single line.

Reply:
I think when a statement fits on a single line, then it should be on a single line. Else, this opens the door to even more subjective decisions to make. But anyway, I can change this one if you want. Ultimately, one should have a program able to parse a statement as a tree (like mmj2 or metamath-knife, but unlike metamath.c) and wrap it according to some uniform rules.

@wlammen, @avekens : what are the requested changes in the present PR ?

Reply:
> I think when a statement fits on a single line, then it should be on a single line. 

I would agree that if the statement including the label and the terminating $= fits on a single line, then it should be on a single line. This does not hold for ~raltpd, so we have a line break anyway. Therefore, we should split the statement in a sensible way, and not after the $p (as demanded also by @wlammen ).